CONTRIBUTING.md

```diff
@@ -0,0 +1,77 @@
+Spotting imperfections and not keeping them to yourself is the first step to make this library better. We are very grateful for reports concerning imperfections in the source code or the [documentation]((https://getstream.io/chat/docs/sdk/react/)). Before filing an issue, please, review the list of [open issues](https://github.com/GetStream/stream-chat-react/issues) first.
+
+### Reporting bugs
+You can report a source code bug by using the [Bug Report template](https://github.com/GetStream/stream-chat-react/issues/new/choose). Make sure you include "steps to reproduce" section. Bug that cannot be reproduced cannot be solved.
+
+Do not be afraid to report imperfections in our [documentation]((https://getstream.io/chat/docs/sdk/react/)) as well. In such case, please attach the `docs` tag to the issue.
+
+
+### Requesting a feature
+You can request a feature by submitting a [Feature request issue](https://github.com/GetStream/stream-chat-react/issues/new?assignees=&labels=feature&template=feature_request.md&title=) in our repository. If you would like to implement the proposal, please state it in the issue. It will allow us to discuss the proposal and better coordinate the efforts. You can even ping us - mention `@GetStream/stream-react-developers ` in the issue.
+
+## <a name="contribution-standards"></a> Contributing to the repo
+
+### Set up for success
+It is always good to get acquainted with the specifics of the package. For example the `stream-chat-react` package has its peer dependencies (`stream-chat-js`, `stream-chat-css`), which you may need to tweak at the same time, while developing the feature for `stream-chat-react`. To get more into those specifics, please read [development guide](./developers/DEVELOPMENT.md).
+
+### Good first issue
+
+###<a name="#coding-rules"></a> Coding rules
+Any contributions to the library should follow Stream's coding rules.
+
+#### 1. Code should be tested
+All the code submitted should be covered by unit tests. Mocking utilities are provided in `src/mock-builders`. Optimally a suite of E2E tests should be included as well.
+
+#### 2. API Changes should be documented
+Changes to components interface exposed to the library integrators should be documented. We keep the documentation `docusaurus/docs/React` folder.
+
+#### 3. Code should be DRY & correctly formatted
+If you find yourself copying source code from one place to another, please extract it into a separate component or function.
+
+#### 4. Keep an eye on performance
+Keep in mind that the chat application may need to work with thousands of messages.
+
+#### 5. Follow commit formatting rules
+We follow [Angular's Commit Message Format rules](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-format) with [possible deviations](./developers/COMMIT.md). The same rules are used by our release automation tool. Therefore, every commit message should strictly comply with these rules.
```

Comment:
```suggestion
We follow [Angular's Commit Message Format rules](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-format) with [possible deviations](./developers/COMMIT.md). The same rules are used by our release automation tool. Therefore, every commit message should strictly comply with these rules. If you're still unsure, you can use [commitizen](https://www.npmjs.com/package/commitizen) as your guide.
```

Reply:
We need to integrate commitlint. React Native has it.

Reply:
Integrating commitlint